src/commands/remove.rs

```diff
@@ -22,29 +22,27 @@ pub async fn remove(
 
     let args = interaction.data.options.clone();
 
-    let remove_index = args
+    let signed_remove_index = args
         .first()
         .unwrap()
         .value
         .as_ref()
         .unwrap()
-        .as_u64()
-        .unwrap() as usize;
+        .as_i64()
+        .unwrap();
 
+    if signed_remove_index.is_negative() || signed_remove_index <= 0 {
+        return create_response(&ctx.http, interaction, "Please provide an index >= 1!").await;
```

Comment:
This is just a nitpick so feel free to disregard, but I think having the inequality in writing (`greater than or equal to`) would be preferable.

Reply:
I think I disagree, but it is not a hard disagree 🤔 Everyone understands math, it's not like the users do not understand a simple `>= 1` in my opinion.